### DISCUSSION TITLE
Focus on "Generate" button

### DISCUSSION THREAD

**Discussion by devkatalist** [SubtitleEdit/subtitleedit#6209] [Ideas]
Hello. For the Audio to text window, could you make it so that the focus is on the button "Generate" so that when this window is launched, we can just press Enter to start generating. (Right now, the focus is on the "Vosk website" link)
![Sans titre](https://user-images.githubusercontent.com/37704802/188262779-d9bfa9ef-5fc7-49c2-bbb1-2d04e00b17fc.png)

**Comment by niksedk** [comment]
Sure :)

**Reply by devkatalist** [reply]
Awesome! Thanks a lot!
